README.md

```diff
@@ -4,3 +4,5 @@ some more text
 to make the 
 file look 
 more full 
+
```

Comment:
Empty line. Please ensure that there are no empty lines to help with computer readibility